defaults/nfs/exports

```diff
@@ -0,0 +1 @@
+/var/log *(ro,root_squash)
```

Comment:
For integralstor, we should exclude this line when we display the list of exports for gridcell, let it be for now..

Reply:
We don't have nfs exports anywhere in the Gridcell views, do we?